### DISCUSSION TITLE
Skip requirements if CivitAi Browser is already installed?

### DISCUSSION THREAD

**Discussion by Terroosh** [BlafKing/sd-civitai-browser-plus#85] [Ideas]
Something I noticed is that SD Webui takes longer to startup when this extension is enabled and this always outputs in console:

`Installing requirements for CivitAI Browser`

Can the requirements check be skipped when it's already installed?

**Comment by BlafKing** [comment]
Thanks for letting me know, I assumed this was caused by SD-WebUI but upon closer inspection it was simply caused by mis-naming the check for if the dependency package was installed, it should no longer attempt to install requirements if they are all met now! :)
